main.js

```diff
@@ -0,0 +1,52 @@
+function getJSON(url, callback) {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', url, false);
+  xhr.send();
+
+  return JSON.parse(xhr.responseText);
+};
+
+let posts = getJSON('https://jsonplaceholder.typicode.com/posts');
```

Comment:
use `const` if you will not change a variable
```suggestion
const posts = getJSON('https://jsonplaceholder.typicode.com/posts');
```